src/Components/GrowthChart/GrowthChartBuilder/GrowthChartBuilder.tsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import i18n from '@dhis2/d2-i18n';
 import { Line } from 'react-chartjs-2';
-import { ChartOptions } from 'chart.js';
-import 'chart.js/auto';
+import Chart, { ChartOptions, CategoryScale } from 'chart.js/auto';
+import annotationPlugin from 'chartjs-plugin-annotation';
```

Comment:
No need to import `CategoryScale`, `Chart` and `annotationPlugin` since we use chart.js version 4